### DISCUSSION TITLE
How can I set management.path_prefix on Windows

### DISCUSSION THREAD

**Discussion by ElanCao** [rabbitmq/rabbitmq-server#2761] [Other] [answered]
Dears,
From this [official page](https://www.rabbitmq.com/management.html) i understand i can set path_prefix for rabbitmq server.
but this guide is for linux, how can i set this param for windows version after i run `rabbitmq-plugins enable rabbitmq_management`?
Thank you guys.

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment, marked as answer]
That guide is in no way Linux-specific. HTTP API [path prefix](https://www.rabbitmq.com/management.html#path-prefix) is set using a [RabbitMQ configuration file key](https://www.rabbitmq.com/configure.html#configuration-files).

**Reply by ElanCao** [reply]
> That guide is in no way Linux-specific. HTTP API [path prefix](https://www.rabbitmq.com/management.html#path-prefix) is set using a [RabbitMQ configuration file key](https://www.rabbitmq.com/configure.html#configuration-files).

thank you, Succeed
